automation-testing/src/main/resources/config.properties

```diff
@@ -1,5 +1,6 @@
 url=https://as.com/meristation/
 url_bank= http://www.bancounion.com.bo/
-ff_driver=webdriver.gecko.driver
-driver_location =C:\\SeleniumDrivers\\geckodriver.exe
+ch= Chrome
+ff= FireFox
+ie= InternetExplorer
```

Comment:
Here, we should have only one variable called "browser" or something similar:
browser = Chrome

The idea is to haven the following line in BaseTest:
this.driver = factory.getDriverType(prop.getPropertyValue("browser"));
This way, we change the "browser" value in config.properties, and not in Basetest.